src/delimiter.js

```diff
@@ -510,22 +585,30 @@ const stackLargeDelimiterSequence = [
 
 /**
  * Get the font used in a delimiter based on what kind of delimiter it is.
+ * TODO(#963) Use more specific font family return type once that is introduced.
  */
-const delimTypeToFont = function(type) {
+const delimTypeToFont = function(type: Delimiter): string {
```

Comment:
Maybe we should limit the return type `"Main-Regular" | "Size1-Regular" | ... | "Size4-Regular"`, wdyt?

Reply:
Nevermind, `flow` will probably have an issue actually verifying the return type.

Reply:
Yea, similar to the previous comment, I'd have to override the compiler. Since this is only used locally, I thought it to not be worth it, but I am open to either change if preferred.